corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -610,7 +611,7 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
 
         clearAction: function () {
             var self = this;
-            this.children.each(function (childView) {
+            this._getChildren().forEach(function (childView) {
```

Comment:
nit: see it was already like this, but can this use `self` instead of `this` here for consistency?